install/docker/src_code_install.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+####### Declare environment variables
+CODE=/code
+BMI3D=$CODE/bmi3d ### Directory in which to install the bmi3d software
+USER=root  # We're in a docker container so root is safe 
+
+####### Download any src code
+git clone https://github.com/sgowda/plot $HOME/code/plotutil
```

Comment:
this repo and the next one shouldn't be needed anymore

Reply:
So should it be safe to just remove then?

Reply:
yes i believe so